Google.PowerShell/CloudResourceManager/IamPolicyCmdlets.cs

```diff
@@ -14,6 +14,52 @@
 
 namespace Google.PowerShell.CloudResourceManager
 {
+    public class GcIamPolicyBindingCmdlet : CloudResourceManagerCmdlet
+    {
+        // IAM Service used for getting roles that can be granted to a project.
+        private Lazy<IamService> iamService =
+            new Lazy<IamService>(() => new IamService(GetBaseClientServiceInitializer()));
+
+        /// <summary>
+        /// Dictionary of roles with key as project and value as the roles available in the project.
+        /// This dictionary is used for caching the various roles available in a project.
+        /// </summary>
+        private static ConcurrentDictionary<string, string[]> s_rolesDictionary =
+            new ConcurrentDictionary<string, string[]>();
+
+        /// <summary>
+        /// Returns all the possible roles that can be granted in a project.
+        /// This is used to provide tab completion for -Role parameter.
+        /// </summary>
+        protected string[] GetGrantableRoles(string projectName)
+        {
+            // We cache the roles in s_rolesDictionary to speed up processing.
+            if (!s_rolesDictionary.ContainsKey(projectName))
+            {
+                var roleRequestBody = new Apis.Iam.v1.Data.QueryGrantableRolesRequest()
+                {
+                    FullResourceName = $"//cloudresourcemanager.googleapis.com/projects/{projectName}"
```

Comment:
I didn't see a reply to the comment for this "should this start with two "/"?"

If not could you add a comment?